AccountingSystem/AccountingSystem/CSVHandler.h

```diff
@@ -34,13 +35,22 @@ CSVHandler<T>::CSVHandler(std::string pathToFile, std::vector<std::string> colum
     : _pathToFile(pathToFile), _columnsName(columnsName), _toInputFormat(convertToInputFormat),
     _toOutputFormat(convertToOutputFormat) {}
 
+template<class T>
+inline CSVHandler<T>* CSVHandler<T>::GetHandler(std::string pathToFile, std::vector<std::string> columnsName, std::function<std::string(T)> convertToInputFormat, std::function<T(std::string)> convertToOutputFormat)
+{
+    std::ofstream myFile(pathToFile);
```

Comment:
What if this file already exists? It seems more proper here would be to do as you did in other functions: check if file exists and create it only if it doesn't